test/parallel/test-http-url.parse-post.js

```diff
@@ -29,7 +29,7 @@ let testURL;
 
 function check(request) {
   // url.parse should not mess with the method
-  assert.strictEqual(request.method, 'POST');
+  assert.strictEqual(request.method, 'POST1');
```

Comment:
Can you revert this change?

Reply:
Ooops, this was done totally by mistake.